tests/gdx-tests/src/com/badlogic/gdx/tests/extensions/ControllersTest.java

```diff
@@ -1,184 +1,283 @@
-/*******************************************************************************
```

Comment:
Sorry, didn't notice this. How can I configure git to do the right thing (via command line or eclipse)?